publisher-sdk/src/main/java/com/criteo/publisher/CriteoBannerView.java

```diff
@@ -41,14 +44,39 @@
    * having to init the SDK before.
    */
   @Nullable
-  private final Criteo criteo;
+  private Criteo criteo;
 
   @Nullable
   private CriteoBannerAdListener criteoBannerAdListener;
 
   @Nullable
   private CriteoBannerEventController criteoBannerEventController;
 
+  /**
+   * Used when setting {@link CriteoBannerView} in XML
+   */
+  public CriteoBannerView(@NonNull Context context, AttributeSet attrs) {
```

Comment:
Shouldn't you need to override the 3 constructors for XML ?
- `constructor(context, attributeSet)`
- `constructor(context, attributeSet, defStyle)`
- `constructor(context, attributeSet, defStyleAttr, defStyleRes)`

Reply:
Only the first one is needed as there is no need to apply custom styling/theming.